### ISSUE TITLE
The primary navigation menus should be navigable using tab

### ISSUE THREAD

**Issue by rautenburger** [swiss/styleguide#551]
A primary navigation menu should be navigable by using the tab key. 
Compare https://q-src.github.io/styleguide/en/ with https://eui.bit.admin.ch/oblique/

When clicking tab, the user this navigates within the primary navigation and not within the menu.

Tested on: 
https://q-src.github.io/styleguide/en/
Tested with:
NVDA screen reader
